source/draw/gpu/GpuCanvasYuv420Semiplanar.ooc

```diff
@@ -45,6 +45,18 @@ GpuCanvasYuv420Semiplanar: class extends GpuCanvas {
 		super(this _target size, context, context defaultMap, IntTransform2D identity)
 		this _target uv canvas pen = Pen new(ColorRgba new(128, 128, 128, 128))
 	}
+	draw: override func ~DrawState (drawState: DrawState) {
+		drawStateY := drawState setTarget((drawState target as GpuYuv420Semiplanar) y)
+		drawStateUV := drawState setTarget((drawState target as GpuYuv420Semiplanar) uv)
+		if (!drawState viewport hasZeroArea)
+			drawStateUV viewport = drawState viewport / 2
+		if (drawState inputImage != null && drawState inputImage class == GpuYuv420Semiplanar) {
+			drawStateY inputImage = (drawState inputImage as GpuYuv420Semiplanar) y
+			drawStateUV inputImage = (drawState inputImage as GpuYuv420Semiplanar) uv
```

Comment:
Give Y and UV from the input image to the different target images if a YUV input image is given.